gaia-fast-list.js

```diff
@@ -0,0 +1,407 @@
+  },
+
+  configure: function(config) {
+    Object.assign(this.userConfig, config);
+    this._createList();
+  },
+
+  attrs: {
+    model: {
+      get: function() { return this._config.model; },
+      set: function(value) {
+        this._config.setModel(value);
+        this._list.reloadData();
+      }
+    }
+  },
+
+  template: `
+    <section class="fast-list">
+      <ul><content></content></ul>
+    </section>
+
+    <style>
+      * { margin: 0; font: inherit; }
+
+      :host {
+        display: block;
+        height: 100%;
+      }
+
+      .fast-list {
+        height: 100%;
+        position: relative;
+      }
+
+      .fast-list ul {
+        list-style: none;
+        padding: 0;
+        margin: 0;
+      }
+
+      ::content h2 {
+        position: sticky;
+        position: -webkit-sticky;
+        top: 0px;
+        height: 32px;
+        z-index: 100;
+
+        margin: 0;
+        padding: 0 6px;
+        box-sizing: border-box;
+        border-bottom: solid 1px var(--border-color);
+        font-size: 0.9em;
+        line-height: 32px;
+
+        background: var(--background-plus);
+        color: var(--title-color);
+      }
+
+      ::content .background {
+        position: absolute;
+        z-index: 0;
+        width: 100%;
+
+        background: linear-gradient(
+          to top,
+          var(--border-color),
+          var(--border-color) 1px,
+          transparent 1px,
+          transparent);
+        background-position: 0 -60px;
+        background-size: 100% 60px;
+        background-repeat: repeat-y;
+      }
+
+      ::content li {
+        -moz-user-select: none;
+        z-index: 10;
+
+        box-sizing: border-box;
+        width: 100%;
+        height: 60px;
+        padding: 9px 16px;
+        border-bottom: solid 1px var(--border-color);
+        font-size: 18px;
+        font-weight: normal;
+        font-style: normal;
+        list-style-type: none;
+        color: var(--text-color);
+      }
+
+      ::content li > a {
+        display: flex;
+        width: 100%;
+        height: 100%;
+        align-items: center;
+        color: inherit;
+        text-decoration: none;
+      }
+
+      ::content > a > div {
+        width: 100%;
+      }
+
+      ::content h3 {
+        margin: 0;
+        font-size: inherit;
+        font-weight: 400;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+
+      ::content p {
+        margin: 0;
+        font-size: 0.7em;
+        line-height: 1.35em;
+      }
+
+      ::content > img {
+        width: 60px;
+        float: right;
+      }
+    </style>`
+});
+
+function Configuration(config) {
+  Object.assign(this, config);
+  this.container = config.container;
+  this.setModel(config.model || []);
+  debug('config init', this);
+}
+
+Configuration.prototype = {
+  setModel: function(model) {
+    this.model = model;
+    this.sections = this.sectionize(this.model);
+  },
+
```

Comment:
let's file a follow up issue to add tests to the component, especially the DataSource/Configuration part